apps/full-site-editing/full-site-editing-plugin/block-patterns/patterns/recent-posts.php

```diff
@@ -0,0 +1,22 @@
+<?php
+/**
+ * Recent Posts pattern.
+ *
+ * @package A8C\FSE
+ */
+
+$markup = '
+<!-- wp:jetpack/layout-grid {"column1DesktopSpan":10,"column1DesktopOffset":1,"column1TabletSpan":8,"column1MobileSpan":4,"column2DesktopOffset":2,"className":"column1-desktop-grid__span-10 column1-desktop-grid__start-2 column1-desktop-grid__row-1 column1-tablet-grid__span-8 column1-tablet-grid__row-1 column1-mobile-grid__span-4 column1-mobile-grid__row-1"} -->
+<div class="wp-block-jetpack-layout-grid alignfull column1-desktop-grid__span-10 column1-desktop-grid__start-2 column1-desktop-grid__row-1 column1-tablet-grid__span-8 column1-tablet-grid__row-1 column1-mobile-grid__span-4 column1-mobile-grid__row-1"><!-- wp:jetpack/layout-grid-column -->
+<div class="wp-block-jetpack-layout-grid-column wp-block-jetpack-layout-grid__padding-none"><!-- wp:a8c/blog-posts {"showAuthor":false,"showCategory":true,"postLayout":"grid","typeScale":3} /--></div>
+<!-- /wp:jetpack/layout-grid-column --></div>
+<!-- /wp:jetpack/layout-grid -->
+';
+
+return array(
+	'__file'  => 'wp_block',
+	'title'   => esc_html__( 'Recent Posts', 'full-site-editing' ),
+	'content' => sprintf(
+		$markup
+	),
```

Comment:
If there's nothing to replace, feel free to pass `$markup` directly
```suggestion
	'content' => $markup,
```